config.py

```diff
@@ -137,14 +147,17 @@ class Live(Config):
 
 class Preview(Live):
     FEATURE_FLAGS_NEW_SUPPLIER_FLOW = enabled_since('2017-02-06')
+    GOOGLE_TAG_MANAGER = dict(Config.GOOGLE_TAG_MANAGER, environmentID='env-7', authToken='Kpjle7g00efQOB2j1o3fVQ')
```

Comment:
Should these actual auth tokens live in the credentials repo rather than a public Github? (And probably be rotated at the same time?)